installer/local_docker/templates/docker-compose.yml.j2

```diff
@@ -0,0 +1,97 @@
+#jinja2: lstrip_blocks: "True", trim_blocks: "True"
```

Comment:
you'll have to further explain this to me why standard ansible templating is not good enough?

Reply:
Hm, your right, some of this is wrong, it comes from another file where I used the same option, but in the meantime, I've realised it was not used correctly, I'll fix it. This being said, I think the standard options won't allow me to keep a proper indentation to make a valid yml file. I'll explore this!

Reply:
trim block is indeed useless here.
lstrib block :
without it, the only way to get
```
    depends_on:
      - rabbitmq
      - memcached
      - postgres
    ports:
      - "80:8052"
```
is if the template is:
```
    depends_on:
      - rabbitmq
      - memcached
      {% if pg_hostname is not defined %}- postgres{% endif %}

    ports:
      - "{{ host_port }}:8052"
```
and if someone were to inadvertently tamper with the empty line before `ports`, that would give:
```
   depends_on:
     - rabbitmq
     - memcached
     - postgres    ports:
     - "80:8052"
```
which would fail.

With `#jinja2: lstrip_blocks: True` the template will be:
```
    depends_on:
      - rabbitmq
      - memcached
      {% if pg_hostname is not defined %}
      - postgres
      {% endif %}
    ports:
      - "{{ host_port }}:8052"
```
which is, imho, quite more readable, and it yields the expected result.

Reply:
Are you sure? I think I've done this same thing without needing the pre-amble.